src/configuration/evars.md

```diff
@@ -285,6 +285,28 @@ If you have set up any [flagzones](http://book.rada.re/basic_commands/flags.html
 
 Show UTF-8 characters instead of ANSI.
 
+### scr.utf8.curvy: `bool`
+
+`scr.utf8.curvy` lets you display curved corners in places like function graphs. This variable requires `scr.utf8` to be turned on.
```

Comment:
I wouldn't repeat the name of the variables here. you can "This variable" or "Use this variable to..." or something like this

Reply:
```suggestion
This variable lets you display curved corners in places like function graphs. This variable requires `scr.utf8` to be turned on.
```